lib/osf-components/addon/components/registries/partial-registration-modal/manager/component.ts

```diff
@@ -0,0 +1,92 @@
+import { action } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import { task } from 'ember-concurrency';
+import DS from 'ember-data';
+import { layout } from 'ember-osf-web/decorators/component';
+import NodeModel from 'ember-osf-web/models/node';
+import defaultTo from 'ember-osf-web/utils/default-to';
+
+import { HierarchicalListManager } from 'osf-components/components/registries/hierarchical-list';
+import template from './template';
+
+@layout(template)
+export default class PartialRegistrationModalManagerComponent extends Component.extend({
+    loadAllChildNodes: task(function *(this: PartialRegistrationModalManagerComponent) {
+        let allChildNodesIncludingRoot = yield this.store.query('node', {
```

Comment:
Just looking at code, I'd expect this to load the first page of nodes, not "all". Am I missing something?

Reply:
Changed to fetch the first page with page size as 100. The current legacy front end doesn't support partial reg for more than 100 nodes.